Source/Plugins/Core/com.equella.core/js/tsrc/schema/SchemaModule.tsx

```diff
@@ -132,36 +130,15 @@ export const pathForNode = (node: SchemaNode, stripXml = true): string => {
  * @param paths Used by the recursive algorithm to build up the returned list.
  *          When passing in initially, leave as a blank array.
  * @param stripXml Passed into pathForNode. Whether to include the leading /xml.
- *
  */
 export const getAllPaths = (
   nodes: SchemaNode,
   paths: string[],
   stripXml = true
```

Comment:
Okay, so keeping paths just for the internal recursive mechanism, how about you swap this around and add a default:

```suggestion
  stripXml = true,
  paths: string[] = []
```

And then typical usage (and your current usage) would just be `getAllPaths(node);` or maybe occassionally `getAllPaths(node, false);` All while still bootstrapping your recursive algorithm.